src/goReferences.ts

```diff
@@ -19,7 +19,7 @@ export class GoReferenceProvider implements vscode.ReferenceProvider {
 		});
 	}
 
-	private doFindReferences(document: vscode.TextDocument, position: vscode.Position, options: { includeDeclaration: boolean }, token: vscode.CancellationToken): Thenable<vscode.Location[]> {
+	public doFindReferences(document: vscode.TextDocument, position: vscode.Position, options: { includeDeclaration: boolean }, token: vscode.CancellationToken): Thenable<vscode.Location[]> {
```

Comment:
Yes, saveAll was the reason for that. Removed `saveAll` and set `provideReferences` back to private